base/deprecated.jl

```diff
@@ -185,6 +185,13 @@ end
 @deprecate iround(x)              round(Integer,x)
 @deprecate iround{T}(::Type{T},x) round(T,x)
 
+# rem1 is inconsistent for x==0: The result should both have the same
```

Comment:
this needs to be in the "0.5 deprecations" section at the end, not mixed in with the 0.4 deprecations